docs/contribute-module.md

```diff
@@ -111,7 +111,7 @@ For more information about testing modules in this project, see [Tests](tests.md
 
 After your PR tests pass, create a pull request.
 
-!> **Important**: Use [Conventional Commit](https://www.conventionalcommits.org) messages when you commit code ([cheat sheet](https://cheatography.com/albelop/cheat-sheets/conventional-commits/)). Conventional commits determine whether a new version of a module is needed and which [semantic versioning number](versioning.md) to use. The commit messages are published in the release notes for the module.
+When the PR is merged, your commit messages might change because the merging team uses the [squash and merge](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges#squash-and-merge-your-pull-request-commits) option to merge PRs. The PR commit message determines whether a new version of a module is needed and which [semantic versioning number](versioning.md) to use. The message also is published in the release notes for the module.
```

Comment:
Perhaps need to more clear here:

`The PR commit message determines whether a new version...` -> `The PR commit message used during the squash and merge determines whether a new version...`

Reply:
how about slightly different: 

The PR commit message `that's included by the merging team` determines whether a new vesion...

> When the PR is merged, your commit messages might change because the merging team uses the squash and merge option to merge PRs. The PR commit message that's included by the merging team determines whether a new version of a module is needed and which semantic versioning number to use. The message also is published in the release notes for the module.

Reply:
fine with me